### PR TITLE
Added note and removed arguments

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.6+ 
Pending QE approval @bgilbert. @miabbott 
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1834830

### CODE REVIEW

File: modules/machine-user-infra-machines-pxe.adoc [hunk 1]
```diff
@@ -27,18 +27,30 @@ TIMEOUT 20
...
 ----
 <1> Specify the location of the live `kernel` file that you uploaded to your HTTP server.
 <2> Specify locations of the {op-system} files that you uploaded to your HTTP server. The `initrd` parameter value is the location of the live `initramfs` file, the `coreos.inst.ignition_url` parameter value is the location of the worker Ignition config file, and the `coreos.live.rootfs_url` parameter value is the location of the live `rootfs` file. The `coreos.inst.ignition_url` and `coreos.live.rootfs_url` parameters only support HTTP and HTTPS.
++
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
+====
```
Comment (ahardin-rh) [inline modules/machine-user-infra-machines-pxe.adoc:39]: see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].

File: modules/machine-user-infra-machines-pxe.adoc [hunk 2]
```diff
@@ -27,18 +27,30 @@ TIMEOUT 20
...
 ----
 <1> Specify locations of the {op-system} files that you uploaded to your HTTP server. The `kernel` parameter value is the location of the `kernel` file, the `coreos.inst.ignition_url` parameter value is the location of the worker Ignition config file, and the `coreos.live.rootfs_url` parameter value is the location of the live `rootfs` file. The `coreos.inst.ignition_url` and `coreos.live.rootfs_url` parameters only support HTTP and HTTPS.
 <2> Specify the location of the `initramfs` file that you uploaded to your HTTP server.
++
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```
Comment (ahardin-rh) [inline modules/machine-user-infra-machines-pxe.adoc:53]: see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].

File: modules/nodes-nodes-kernel-arguments.adoc [hunk 3]
```diff
@@ -162,11 +162,16 @@ Starting pod/ip-10-0-141-105ec2internal-debug ...
...
 coreos.oem.id=qemu coreos.oem.id=ec2 ignition.platform.id=ec2 selinux=0
 
 sh-4.2# exit
 ----
 +
 You should see the `selinux=0` argument added to the other kernel arguments.
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```
Comment (ahardin-rh) [inline modules/nodes-nodes-kernel-arguments.adoc:176]: see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].

File: modules/machine-user-infra-machines-pxe.adoc [hunk 4]
```diff
@@ -27,18 +27,30 @@ TIMEOUT 20
...
 ----
 <1> Specify the location of the live `kernel` file that you uploaded to your HTTP server.
 <2> Specify locations of the {op-system} files that you uploaded to your HTTP server. The `initrd` parameter value is the location of the live `initramfs` file, the `coreos.inst.ignition_url` parameter value is the location of the worker Ignition config file, and the `coreos.live.rootfs_url` parameter value is the location of the live `rootfs` file. The `coreos.inst.ignition_url` and `coreos.live.rootfs_url` parameters only support HTTP and HTTPS.
++
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`. This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```
Comment (bgilbert) [inline modules/machine-user-infra-machines-pxe.adoc:38]: Hmm, this version includes both the original text and my rewrite, which makes for some extra redundancy and a difficult-to-follow subject shift after the second sentence.  I had intended for my text to replace the existing text.  Can we consolidate the two versions?

File: modules/nodes-nodes-kernel-arguments.adoc [hunk 5]
```diff
@@ -162,11 +162,16 @@ Starting pod/ip-10-0-141-105ec2internal-debug ...
 To use host binaries, run `chroot /host`
 
 sh-4.2# cat /host/proc/cmdline
-BOOT_IMAGE=/ostree/rhcos-... console=tty0 console=ttyS0,115200n8
+BOOT_IMAGE=/ostree/rhcos-... ,115200n8
```
Comment (bgilbert) [inline modules/nodes-nodes-kernel-arguments.adoc:165]: `,115200n8` is part of the `console=` argument and should be removed too.

File: modules/machine-user-infra-machines-pxe.adoc (hunk 2 above)
Comment (bgilbert) [reply modules/machine-user-infra-machines-pxe.adoc:53]: ...add one or more `console=` arguments to the `kernel` line.

File: modules/nodes-nodes-kernel-arguments.adoc (hunk 3 above)
Comment (bgilbert) [reply modules/nodes-nodes-kernel-arguments.adoc:176]: ...add one or more `console=` arguments to the `kernelArguments` section.

File: modules/nodes-nodes-kernel-arguments.adoc (hunk 5 above)
Comment (bgilbert) [reply modules/nodes-nodes-kernel-arguments.adoc:165]: Whoops, https://github.com/openshift/openshift-docs/pull/29525#discussion_r578793753 is correct.  Disregard.

File: modules/nodes-nodes-kernel-arguments.adoc (hunk 3 above)
Comment (bgilbert) [reply modules/nodes-nodes-kernel-arguments.adoc:176]: Whoops, https://github.com/openshift/openshift-docs/pull/29525#discussion_r578793949 is correct.  Disregard my comment.

**Comment by bgilbert** [comment]
Left comments in https://github.com/openshift/openshift-docs/pull/29525#pullrequestreview-594515289.  Also, this version seems to have lost the NOTE addition from `installation-user-infra-machines-pxe.adoc`.

**Comment by sagidlow** [comment]
> Left comments in [#29525 (review)](https://github.com/openshift/openshift-docs/pull/29525#pullrequestreview-594515289). Also, this version seems to have lost the NOTE addition from `installation-user-infra-machines-pxe.adoc`.



> Left comments in [#29525 (review)](https://github.com/openshift/openshift-docs/pull/29525#pullrequestreview-594515289). Also, this version seems to have lost the NOTE addition from `installation-user-infra-machines-pxe.adoc`.

@bgilbert  that file didn't exist in the master branch for 4.6 +

**Comment by bgilbert** [comment]
[This one](https://github.com/openshift/openshift-docs/blob/master/modules/installation-user-infra-machines-pxe.adoc)?

**Comment by sagidlow** [comment]
@bgilbert it doesn't mention `console=ttyS0`, so I wasn't able to find it. Do you want me to add the note for PXE and iPXE instructions?

**Comment by bgilbert** [comment]
Yup, we removed the `console=` arguments from that file in 4.6, but it still needs the NOTE.

**Review by bgilbert — APPROVED** [review]

**Comment by sagidlow** [comment]
@ahardin-rh - Ready for Merge.

**Pull request merged**